index.js

```diff
@@ -71,11 +71,16 @@ module.exports.templateTags = [{
             type: 'string',
             placeholder: 'MMMM Do YYYY, h:mm:ss a',
             hide: args => args[8].value !== 'custom'
+        },
+        {
+            displayName:'Locale',
+            type:'string',
```

Comment:
Make this type 'enum' and fill it with available locales from the method `moment.locales()`